Nearest neighbor interpolation fix

JimNrao opened the pull request:
Fixed nearestNeigbour interpolation to properly handle an abscissa sorted in descending order.

Mods by George Moellenbrock.

gervandiepen commented:
There were many more file changes than the interpolation. How can?

On Thu, Aug 18, 2016 at 11:30 PM, Jim <notifications@github.com> wrote:

> Closed #484 <https://github.com/casacore/casacore/pull/484>.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/casacore/casacore/pull/484#event-760537755>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AHfIPJpwkEKr7wpRGaGtgianl4t_coayks5qhM7vgaJpZM4Jn7yZ>
> .
>

tammojan commented:
This pull request was closed, probably for that reason. Have a look at #485 instead.

JimNrao commented:
I thought I had updated my local repo prior to applying the mod but 
apparently didn't merge it into the code.  So, I killed the PR, properly 
updated the code and issued a new PR.  I haven't been using git enough 
to get really solid with it and haven't used it much at all for several 
months.


On 08/19/2016 01:58 AM, Tammo Jan Dijkema wrote:
>
> This pull request was closed, probably for that reason. Have a look at 
> #485 <https://github.com/casacore/casacore/pull/485> instead.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub 
> <https://github.com/casacore/casacore/pull/484#issuecomment-240953433>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AGPGtNa-uT7Ogh3ieWDwdq9CKhlJnPKeks5qhWIIgaJpZM4Jn7yZ>.Web 
> Bug from 
> https://github.com/notifications/beacon/AGPGtL64USMcrygRV5jNW8LR-yVFalKGks5qhWIIgaJpZM4Jn7yZ.gif
>
> {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/casacore/casacore","title":"casacore/casacore","subtitle":"GitHub 
> repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open 
> in 
> GitHub","url":"https://github.com/casacore/casacore"}},"updates":{"snippets":[{"icon":"PERSON","message":"@tammojan 
> in #484: This pull request was closed, probably for that reason. Have 
> a look at #485 instead."}],"action":{"name":"View Pull 
> Request","url":"https://github.com/casacore/casacore/pull/484#issuecomment-240953433"}}}

-- 
Jim Jacobs

phone: 835-7235
office: 301
